crates/types/src/services/p2p.rs

```diff
@@ -69,3 +70,11 @@ impl<T: Debug + Send + 'static> NetworkData<T> for GossipData<T> {
         self.data.take()
     }
 }
+/// Used for relying latest `BlockHeight` info from connected peers
+#[derive(Debug, Clone)]
+pub struct BlockHeightHeartbeatData {
+    /// PeerId as bytes
+    pub peer_id: Vec<u8>,
```

Comment:
We can't send the peer ids in the heartbeat because imagine the scenario where the sync look has commit header `4` then get's the heartbeat for `8`. What peer id will be associated with `5, 6, 7`?